### ISSUE TITLE
RockMiner (aside from orig. R-Box) driver performs poorly

### ISSUE THREAD

**Issue by nwoolls** [luke-jr/bfgminer#564]
The RockMiner driver performs at about 3/4 the speed of CGMiner, both official and forked. This is confirmed server-side with both the R-3 Box and the "NEW" R-Box.

BFGMiner @ 310mhz:
![bfgminer new rbox mac](https://cloud.githubusercontent.com/assets/1683618/4514657/189c7d2e-4b82-11e4-8bd7-2f8d239441bf.png)

CGMiner @ 270mhz:
![cgminer new rbox mac](https://cloud.githubusercontent.com/assets/1683618/4514658/1d2dd84c-4b82-11e4-82f2-f1a1c24b02be.png)

**Comment by luke-jr** [comment]
I have nothing to go on...

**Comment by nwoolls** [comment]
Just wanted an official place to start. What do you need to know and I'll be happy to try to get more information.

This is easily reproducible with both the R3-Box and the New R-Box. Is it hardware / protocol documentation you need?

**Comment by luke-jr** [comment]
Right. I was emailing with Alex Lam, but he stopped responding about 2 weeks ago before I had anything to work with. :(

**Comment by nwoolls** [comment]
Gotcha - unfortunately that's the same fellow I talk to but I'll see what I can do.

**Comment by imnotrich** [comment]
Running Ubuntu Studio 14.10 64bit, bfgminer 4.7.0 (the most recent version I can find compiled for this OS) and a Rockminer New R-Box connecting through a powered pci-e internal usb 3.0 hub from Inatek. Using slush's pool. Setting the frequency at 270 I'm lucky to see 75Ghs on a device advertised at 100-100. Setting the frequcy for 280-290 or 300 results in many errors per hour and maybe gets me 80-85Ghs. Using the unsafe frequencies above 300 gets me hundreds of hardware errors per hour and no increase in speeds. The Rockminer is cool to the touch. I have also an external powered hub with some usb antminers and a block eruptor if that matters - have not tried running the Rockminer alone yet.  Please tell me how to grab the log files or whatever other info you need, big time noob here my tech skills are limited but I'm learning. Maybe trying to install bfgminer 5.1 though I don't know how to do that since I can't find a deb or ppa yet for 5.1. Thanks!
